app/views/spree/users/edit.html.haml

```diff
@@ -1,3 +1,4 @@
 .darkswarm
   .row
+  .user-form
```

Comment:
you can have a single first line
`.darkswarm.user-form`
I think that will be equivalent to your first attempt

then the style will be:
.darkswarm.user-form (no space between them)

Reply:
I attempted to change it to `.darkswarm.user-form` but it did not seem to properly apply the `.user-form` style to the page so I left it as separate lines in my more recent commits.